json-endpoints/jetpack/json-api-jetpack-endpoints.php

```diff
@@ -902,3 +902,111 @@
 		),
 	),
 ) );
+
+
+require_once( $json_jetpack_endpoints_dir . 'class.jetpack-json-api-cron-endpoint.php' );
+
+// GET /sites/%s/cron
+new Jetpack_JSON_API_Cron_Endpoint( array(
+	'description'     => 'Fetches the cron array',
+	'group'           => '__do_not_document',
+	'method'          => 'GET',
+	'path'            => '/sites/%s/cron',
+	'stat'            => 'cron-get',
+	'path_labels' => array(
+		'$site' => '(int|string) The site ID, The site domain'
+	),
+	'response_format' => array(
+		'cron_array' => '(array) The cron array',
+		'current_timestamp' => '(int) Current server timestamp'
+	),
+	'example_request' => 'https://public-api.wordpress.com/rest/v1.1/sites/example.wordpress.org/cron',
+	'example_request_data' => array(
+		'headers' => array( 'authorization' => 'Bearer YOUR_API_TOKEN' ),
+	),
+) );
+
+// POST /sites/%s/cron
+new Jetpack_JSON_API_Cron_Post_Endpoint( array(
+	'description'     => 'Process items in the cron',
+	'group'           => '__do_not_document',
+	'method'          => 'POST',
+	'path'            => '/sites/%s/cron',
+	'stat'            => 'cron-run',
+	'path_labels' => array(
+		'$site' => '(int|string) The site ID, The site domain'
+	),
+	'request_format' => array(
+		'hooks'       => '(array) List of hooks to run if they have been scheduled',
+	),
+	'response_format' => array(
+		'success' => '(array) Of processed hooks with their arguments'
+	),
+	'example_request' => 'https://public-api.wordpress.com/rest/v1.1/sites/example.wordpress.org/cron',
+	'example_request_data' => array(
+		'headers' => array( 'authorization' => 'Bearer YOUR_API_TOKEN' ),
+		'body' => array(
+			'hooks'   => array( 'jetpack_sync_cron' )
+		),
+	),
+) );
+
+// POST /sites/%s/cron/schedule
+new Jetpack_JSON_API_Cron_Schedule_Endpoint( array(
+	'description'     => 'Process items in the cron',
+	'group'           => '__do_not_document',
+	'method'          => 'POST',
+	'path'            => '/sites/%s/cron/schedule',
+	'stat'            => 'cron-schedule',
+	'path_labels' => array(
+		'$site' => '(int|string) The site ID, The site domain'
+	),
+	'request_format' => array(
+		'hook'             => '(string) Hook name that should run when the event is scheduled',
+		'arguments'        => '(string) JSON Object of arguments that the hook will use',
```

Comment:
Should we add that arguments is optional since we're checking if it's set in the endpoint?